CHANGELOG.md

```diff
@@ -75,5 +76,5 @@
 ## 1.1.0 - 2019-03-19
 
 ### Added
-- Added possibility to specify charset for regular and native ingesters.
+- Added possibility to specify charset for regular and native ingester.
```

Comment:
Not sure why not use the previous plural, since it's related to the 2 types (regular/native).

Reply:
Mmmmm, maybe you are right, reverted.